### ISSUE TITLE
/metrics container_fs_reads_total is 0

### ISSUE THREAD

**Issue by pangzheng** [google/cadvisor#1669]
I can not get the io information for the docker instance～

```
docker stats 

CONTAINER                    CPU %               MEM USAGE / LIMIT       MEM %               NET I/O             BLOCK I/O             PIDS
prometheus   3.12%               4.658 GiB / 7.642 GiB   60.95%              0 B / 0 B           11.39 GB / 238.6 GB   0

cat /sys/fs/cgroup/blkio/docker/e0b99e597af44848583846fe328a4c388cf17a8ce56d6f5c98b5b8f2fee27d30/blkio.throttle.io_service_bytes
253:1 Read 0
253:1 Write 348160
253:1 Sync 0
253:1 Async 348160
253:1 Total 348160
8:0 Read 5709284352
8:0 Write 119474752000
8:0 Sync 119474379264
8:0 Async 5709657088
8:0 Total 125184036352
253:0 Read 5709284352
253:0 Write 119474379264
253:0 Sync 119474379264
253:0 Async 5709284352
253:0 Total 125183663616
Total 250368048128

docker api 
 curl 172.16.1.9:2375/containers/1e2c455a16a30f61bb3bfdf55a215337a09235ce06bd282ef36cf82ca4e0a817/stats
{"read":"2017-06-07T04:04:41.686296709Z","precpu_stats":{"cpu_usage":{"total_usage":0,"percpu_usage":null,"usage_in_kernelmode":0,"usage_in_usermode":0},"system_cpu_usage":0,"throttling_data":{"periods":0,"throttled_periods":0,"throttled_time":0}},"cpu_stats":{"cpu_usage":{"total_usage":451979102820,"percpu_usage":[231613721917,220365380903],"usage_in_kernelmode":140730000000,"usage_in_usermode":273590000000},"system_cpu_usage":350885860000000,"throttling_data":{"periods":0,"throttled_periods":0,"throttled_time":0}},"memory_stats":{"usage":23240704,"max_usage":52277248,"stats":{"active_anon":19931136,"active_file":1048576,"cache":3309568,"hierarchical_memory_limit":9223372036854775807,"hierarchical_memsw_limit":9223372036854775807,"inactive_anon":0,"inactive_file":2260992,"mapped_file":0,"pgfault":38029,"pgmajfault":0,"pgpgin":34270,"pgpgout":29618,"rss":19931136,"rss_huge":0,"swap":0,"total_active_anon":19931136,"total_active_file":1048576,"total_cache":3309568,"total_inactive_anon":0,"total_inactive_file":2260992,"total_mapped_file":0,"total_pgfault":38029,"total_pgmajfault":0,"total_pgpgin":34270,"total_pgpgout":29618,"total_rss":19931136,"total_rss_huge":0,"total_swap":0,"total_unevictable":0,"unevictable":0},"failcnt":0,"limit":8205836288},"blkio_stats":{"io_service_bytes_recursive":[{"major":8,"minor":0,"op":"Read","value":3162112},{"major":8,"minor":0,"op":"Write","value":0},{"major":8,"minor":0,"op":"Sync","value":0},{"major":8,"minor":0,"op":"Async","value":3162112},{"major":8,"minor":0,"op":"Total","value":3162112},{"major":253,"minor":0,"op":"Read","value":3162112},{"major":253,"minor":0,"op":"Write","value":0},{"major":253,"minor":0,"op":"Sync","value":0},{"major":253,"minor":0,"op":"Async","value":3162112},{"major":253,"minor":0,"op":"Total","value":3162112}],"io_serviced_recursive":[{"major":8,"minor":0,"op":"Read","value":25},{"major":8,"minor":0,"op":"Write","value":0},{"major":8,"minor":0,"op":"Sync","value":0},{"major":8,"minor":0,"op":"Async","value":25},{"major":8,"minor":0,"op":"Total","value":25},{"major":253,"minor":0,"op":"Read","value":25},{"major":253,"minor":0,"op":"Write","value":0},{"major":253,"minor":0,"op":"Sync","value":0},{"major":253,"minor":0,"op":"Async","value":25},{"major":253,"minor":0,"op":"Total","value":25}],"io_queue_recursive":[],"io_service_time_recursive":[],"io_wait_time_recursive":[],"io_merged_recursive":[],"io_time_recursive":[],"sectors_recursive":[]},"pids_stats":{}}
```

version infomation

```
# cadvisor -version
cAdvisor version v0.25.0 (17543be)
```

```
# docker info
Containers: 11
 Running: 11
 Paused: 0
 Stopped: 0
Images: 18
Server Version: 1.12.1
Storage Driver: overlay
 Backing Filesystem: xfs
Logging Driver: none
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge null host overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 3.10.0-327.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 7.642 GiB
Name: srymaster3
ID: KXTE:ZL4B:SKNA:MHQR:ESVK:6526:EUPF:WX3W:4SGE:KJQ2:AK7E:EKZN
Docker Root Dir: /data/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Insecure Registries:
 172.16.1.11
```

Hope to get a reply, thanks

**Comment by pangzheng** [comment]
cadvisor api has data

diskio: {
io_service_bytes: [
{
major: 253,
minor: 0,
stats: {
Async: 6170981376,
Read: 6170981376,
Sync: 122608486912,
Total: 128779468288,
Write: 122608486912
}
},
{
major: 253,
minor: 1,
stats: {
Async: 348160,
Read: 0,
Sync: 0,
Total: 348160,
Write: 348160
}
},
{
major: 8,
minor: 0,
stats: {
Async: 6171354112,
Read: 6170981376,
Sync: 122608486912,
Total: 128779841024,
Write: 122608859648
}
}
],
io_serviced: [
{
major: 253,
minor: 1,
stats: {
Async: 85,
Read: 0,
Sync: 0,
Total: 85,
Write: 85
}
},
{
major: 8,
minor: 0,
stats: {
Async: 1404026,
Read: 1403935,
Sync: 4360625,
Total: 5764651,
Write: 4360716
}
},
{
major: 253,
minor: 0,
stats: {
Async: 1403935,
Read: 1403935,
Sync: 4360625,
Total: 5764560,
Write: 4360625
}
}
]
},

**Comment by dashpole** [comment]
I think this is fixed in the most recent release (v0.26.0) by #1642.  I havent had a chance to build and upload the latest version, but you can build from source to see if that fixes your issues.
